docfx/doxygen2syntax.cc

```diff
@@ -81,6 +82,21 @@ std::string LinkedTextType(pugi::xml_node const& node) {
   return std::move(os).str();
 }
 
+std::string HtmlEscape(std::string_view text) {
+  std::string clean;
+  clean.reserve(text.size());
```

Comment:
If you're going to bother with this at all, should it include some allowance for expansion?

Reply:
I could not convince myself that any allowance was particularly good.  Removed the `.reserve()` call.